player/command.c

```diff
@@ -6233,6 +6234,26 @@ static void cmd_apply_profile(void *p)
     }
 }
 
+static void cmd_load_config_file(void *p)
+{
+    struct mp_cmd_ctx *cmd = p;
+    struct MPContext *mpctx = cmd->mpctx;
+
+    char *config_file = cmd->args[0].v.s;
+    int r = m_config_parse_config_file(mpctx->mconfig, mpctx->global,
+                                       config_file, NULL, 0);
+
+    if (r < 1) {
+        cmd->success = false;
+        return;
+    }
+
+    mp_notify_property(mpctx, "profile-list");
+
+    if (mpctx->video_out)
+        vo_redraw(mpctx->video_out);
```

Comment:
What's this needed for? Is it because new values from the new config don't get updated or something?

Reply:
Yes, it updates properties like gamma before unpausing.

Reply:
Hmm feels like a workaround and it might not cover everything like this. Is there not a way to force the option change callback to run or something?

Reply:
`mp_option_change_callback` is already called but a redraw is still necessary to update the OSD.

Reply:
This is weird, but it works without manually redrawing after adding `mp_notify_property(mpctx, "profile-list");`. The redraw was necessary before I added that line, though it would still randomly update the VO immediately.

Reply:
huh, not what I would have expected either but that works for me.